.github/workflows/e2e.yaml

```diff
@@ -32,7 +28,13 @@ jobs:
       - name: Setup Helm
         uses: fluxcd/pkg/actions/helm@main
       - name: Run tests
-        run: make test
+        run: |
+          docker run --network host \
+                     --volume $(pwd):$(pwd) \
+                     --volume ~/go/pkg/mod:/root/go/pkg/mod \
+                     --env KUBEBUILDER_ASSETS=$KUBEBUILDER_ASSETS \
+                     --workdir $(pwd) golang:1.15-alpine /bin/sh -c \
+                     "apk add --no-cache build-base git ; make test"
         env:
           KUBEBUILDER_ASSETS: ${{ github.workspace }}/kubebuilder/bin
       - name: Check if working tree is dirty
```

Comment:
It is not necessary to run this in the container as we're mirror mounting the workdir into the container.